include/ua_plugin_nodestore.h

```diff
@@ -22,6 +22,13 @@ extern "C" {
 #endif
 
 #include "ua_server.h"
+#ifdef UA_ENABLE_EVENTS
+#include <queue.h>
```

Comment:
This is the first time we expose queue.h in the public API.
That can be problematic for amalgamation. If the user include queue.h and open62541.h
Can we make this a singly-linked list with a "manual" next-pointer?